examples/with-cookie-auth/package.json

```diff
@@ -1,5 +1,15 @@
 {
   "name": "with-cookie-auth",
-  "version": "1.0.0",
-  "description": "Monorepo with a Next.js example of cookie based authorization"
+  "scripts": {
+    "build": "next build",
```

Comment:
You can now add `"dev": "next",` too, it should work because everything is in a single app.